modules/op-pipelines-concepts.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * openshift_pipelines/understanding-openshift-pipelines.adoc
+
+[id='op-pipelines-concepts_{context}']
+= {op-title} concepts
```

Comment:
Missing `Condition` and trigger objects

Reply:
done, request you to kindly review. Thank you :)

Reply:
@siamaksade ^^